### DISCUSSION TITLE
I have a problem in using one chain

### DISCUSSION THREAD

**Discussion by aoPillz** [wevm/wagmi#2677] [Q&A]
Error: Unsupported chains. connect() optionalNamespaces, chains must be defined as "namespace:chainId" e.g. "eip155:1": {...} in the namespace key OR as an array of CAIP-2 chainIds e.g. eip155: { chains: ["eip155:1", "eip155:5"] }

When I want to use only mainnet, an error occurs, but when I add 1 more, there is no error, how can I make it work for one chain?

My code:

import { WagmiConfig, configureChains, createConfig, mainnet } from "wagmi";
import { WalletConnectConnector } from "wagmi/connectors/walletConnect";
import { infuraProvider } from "wagmi/providers/infura";

const { publicClient } = configureChains(
  [mainnet],
  [
    infuraProvider({
      apiKey: "...",
      rpcUrl: "...",
    }),
  ],
);

const config = createConfig({
  autoConnect: true,
  connectors: [
    new WalletConnectConnector({
      chains: [mainnet],
      options: {
        projectId: "...",
      },
    }),
  ],
  publicClient,
});

**Comment by Max20397** [comment]
same error :(
